ccpp/data/GFS_typedefs.F90

```diff
@@ -337,6 +337,9 @@ module GFS_typedefs
 
 ! -- In/Out for Noah MP
     real (kind=kind_phys), pointer :: snowxy  (:)  => null()  !<
+    real (kind=kind_phys), pointer :: dzsno   (:)  => null()  !<
+    real (kind=kind_phys), pointer :: dzsnso  (:)  => null()  !<
+
```

Comment:
it's not clear why you need these variables. is this just temporary? they are 1d which I guess just means they can be used for one location only?